recipes/r-recetox-waveica/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "recetox-waveica" %}
+{% set version = "0.1.0" %}
+{% set github = "https://github.com/recetox/waveica" %}
+
+package:
+  name: "r-{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "{{ github }}/archive/v{{ version }}.tar.gz"
+  sha256: 5d6ca5d6bc95b1c169a58afb76810dbaf169c58f190f6f8969c0cd2d83535743
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/
+    - lib/R/lib/
+  script: R CMD INSTALL --build .
+
+requirements:
+  host:
+    - r-base >=3.5.0
+    - r-waveslim
+    - r-jade
+    - r-corpcor
+    - r-proc
+    - r-plsdepot
+    - r-fdrtool
+    - r-scatterplot3d
+    - r-ggfortify
+  run:
+    - r-base
+    - r-waveslim
+    - r-jade
+    - r-corpcor
+    - r-proc
+    - r-plsdepot
+    - r-fdrtool
+    - r-scatterplot3d
+    - r-ggfortify
+    - r-ggplot2
+    - r-rcolorbrewer
+
+test:
+  commands:
+    - $R -e "library('WaveICA')"
+
+about:
+  home: "{{ github }}"
+  license: MIT
+  summary: Removal of batch effects for large-scale untargeted metabolomics data based on wavelet analysis.
+
+extra:
+  maintainers:
+    - RECETOX/conda
```

Comment:
We've tested this and it sadly does not work, the teams are invisible/not pingable from outside of the organisation. These have to be actual personal accounts.

Reply:
I see. That's a pity. Updated the maintainers in af65299.